code/__HELPERS/text/format.dm

```diff
@@ -15,4 +15,4 @@
 		return
 	// todo: optimize this if needed, maybe dynamic regex?
 	for(var/key in keys)
-		. = replacetext_char(., "$[key]", keys[key])
+		. = replacetext_char(., "%%[key]%%", keys[key])
```

Comment:
it was either this or `${THIS}`; kinda leaning to the latter but not hard to regex for it later

`$` is simply too common to use as a replacement.